editor/src/components/editor/actions/wrap-unwrap-helpers.tsx

```diff
@@ -159,9 +158,7 @@ export function unwrapTextContainingConditional(
     (success) => {
       if (elementToInsert != null) {
         const components = getUtopiaJSXComponentsFromSuccess(success)
-        const updatedComponents = insertElementAtPath_DEPRECATED(
-          editor.projectContents,
-          editor.canvas.openFile?.filename ?? null,
+        const updatedComponents = insertElementAtPath(
           childInsertionPath(targetParent),
```

Comment:
is this right? what if the targetParent is a conditional?

Reply:
it seems to be working fine, see https://github.com/concrete-utopia/utopia/commit/c66cfaee959183ab8f80ff129bce8be958594938